src/app/csv-collection/csv-collection-detail/csv-collection-detail.component.ts

```diff
@@ -189,15 +194,7 @@ export class CsvCollectionDetailComponent implements OnInit, AfterViewInit, OnDe
   }
 
   getCsvFiles(event): void {
-    const sortField = event?.sortField ? event.sortField : 'fileName,asc';
-    const pageIndex = event ? event.first / event.rows : 0;
-    const pageSize = event ? event.rows : this.pageSize;
-    const params = {
-      pageIndex: pageIndex,
-      size: pageSize,
-      sort: sortField
-    };
-    this.csvCollectionService.getCsvFiles(this.csvCollection, params).subscribe(paginatedResult => {
+    this.csvCollectionService.getCsvFiles(this.csvCollection/*, params*/).subscribe(paginatedResult => {
```

Comment:
Params are needed for pagination (even if we remove the sorting options), otherwise we always get the first page